coderd/database/migrations/000363_workspace_build_initiator_index.up.sql

```diff
@@ -0,0 +1,6 @@
+-- Add index on workspace_builds.initiator_id to optimize prebuild queries
+-- This will dramatically improve performance for:
+-- - GetPrebuildMetrics (called every 15 seconds)
```

Comment:
This is stale now, no?

```suggestion
-- - GetPrebuildMetrics (called every 60 seconds) (see metricsUpdateInterval in enterprise/coderd/prebuilds/metricscollector.go).
```